### ISSUE TITLE
SSO: Tooltip and tooltip textbox display enhancements for consistency and greater accessibility

### ISSUE THREAD

**Issue by coder-karen** [Automattic/jetpack#37674]
### Impacted plugin

Jetpack

### What

SSO tooltip texboxes should have no visibility issues (making them more accessible), and tooltips consistent

### How

The textboxes themselves should ideally have a more obviously different background color or a border if possible - something to help differentiate between the backgound color and the text box itself.

The tooltip icons, if possible, would ideally be the same.

Steps to replicate: 

* Start with a site that's connected to WordPress.com, and where you have activated the SSO feature under Jetpack > Settings > Security
* Go to Users > All Users in your dashboard.
* Make sure you have at least 2 users, one of which has not been invited to join WordPress.com when creating the user.

Current display:
<img width="337" alt="Screenshot 2024-06-03 at 09 35 47" src="https://github.com/Automattic/jetpack/assets/16754605/7440bc01-5173-4e2e-a01a-a969171c226a">

This was suggested in internal testing: p8oabR-1wt-p2#comment-8015
